scanner/src/test/resources/oxfordnanopore/promethion/20190221_2053_2-E3-H3_PAD10721_98c289ed/reference.json

```diff
@@ -0,0 +1 @@
+{"platform":"OxfordNanopore","runAlias":"20190221_2053_2-E3-H3_PAD10721_98c289ed","sequencerFolderPath":null,"sequencerName":"unittest","sequencerPosition":"2-E3-H3","containerSerialNumber":"PAD10721","containerModel":"flo-pro002","laneCount":1,"healthType":"UNKNOWN","startDate":[2019,2,21,20,54],"completionDate":null,"pairedEndRun":false,"software":null,"metrics":null,"runType":"sequencing_run"}
```

Comment:
We've been putting all of these throught `jq -S .` to format them and to minimise diffs when we update them in the future.